api/v1alpha1/common_types.go

```diff
@@ -162,3 +170,20 @@ type ClusterStatusConfig struct {
 	// WorkerSelector is the `scale.status.selector` subresource field.
 	WorkerSelector string `json:"workerSelector,omitempty"`
 }
+
+const (
+	PendingStatus  batchv1.JobConditionType = "Pending"
```

Comment:
I know this was already defined, but this type seems wrong. Its an alias for a `string`, but only some of these enum variants are actually defined. So IDK if there is value in using this type.

Reply:
Yeah, I think there should be a separate (our own) custom type based on string.